### DISCUSSION TITLE
Git error while trying with snowflake

### DISCUSSION THREAD

**Discussion by ravilakkireddy** [theory/sqitch#637] [Q&A]
I am trying to use sqitch for snowflake and following the https://sqitch.org/docs/manual/sqitchtutorial-snowflake/.
When I run this command , I am getting the below error

$ sqitch init flipr --uri https://github.com/sqitchers/sqitch-snowflake-intro/ --engine snowflake
bash: sqitch: command not found

how to resolve this?

**Comment by theory** [comment]
It looks like you do not have Sqitch installed, or perhaps not in your path. Have you installed it, and if so how? Or are you using a Docker image?
